lib/swaps/Swaps.ts

```diff
@@ -636,6 +658,12 @@ class Swaps extends EventEmitter {
 
   }
 
+  private handleSwapTimeout = (rHash: string) => {
+    const deal = this.getDeal(rHash)!;
+    this.timeouts.delete(rHash);
+    this.failDeal(deal, SwapFailureReason.Timeout);
```

Comment:
perhaps create a different failure reason to differ between acceptance/completion timeout